src/poetry/repositories/cached_repository.py

```diff
@@ -22,7 +22,7 @@
 
 
 class CachedRepository(Repository, ABC):
-    CACHE_VERSION = parse_constraint("1.1.0")
+    CACHE_VERSION = parse_constraint("2.0.0")
```

Comment:
Could you add info about this to your original PR description? It would make it easier for us to put an info about this in release notes (I assume people may want to remove their old caches)

Reply:
I wouldn't expect to need to spell this out in the release note.  

The last time this was changed was with the addition of support for yanked releases, it wasn't called out then.

Reply:
Fine then.